packages/core/src/loaders/configuration/interleavedRetrieve.ts

```diff
@@ -87,10 +87,5 @@ const interleavedRetrieveConfiguration: RetrieveStage[] = [
     requestType: RequestType.Thumbnail,
     retrieveType: 'multipleFinal',
   },
-  {
```

Comment:
Can you leave this commented out - I need to fix things rather than removing it, because there are real situations where it needs to fallback to the default retrieval.